### PR TITLE
Replace cdn.ampproject.org with ampjs.org in various hard-coded places around the codebase

### PR DESCRIPTION
**Pull request by danielrozenberg**
I tried to avoid making changes to where it looks like cdn.ampproject.org is used as a cache and not a runtime server, and didn't make changes to places where it indicates which runtime server is the "default". I obviously missed some parts here and there so let's give this an extra through review

(draft PR for now, I'll mark this ready after the validator change)

### CODE REVIEW

File: extensions/amp-ad-network-adzerk-impl/0.1/data/0 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "templateUrl": "https://cdn.ampproject.org/ad-template/0001.html",
+  "templateUrl": "https://ampjs.org/ad-template/0001.html",
```
Comment (jridgewell) [inline extensions/amp-ad-network-adzerk-impl/0.1/data/0:2]: Is this hosted by `ampjs.org`?

File: src/config.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ export const urls = {
   thirdPartyFrameHost: env['thirdPartyFrameHost'] || 'ampproject.net',
   thirdPartyFrameRegex,
   cdn:
+    // TODO(danielrozenberg): replace default hostname with ampjs.org.
     env['cdnUrl'] || getMetaUrl('runtime-host') || 'https://cdn.ampproject.org',
```
Comment (jridgewell) [inline src/config.js:69]: I think we should be creating a second variable, one for JS URLs and one for Cache URLs.

File: extensions/amp-ad-network-adzerk-impl/0.1/data/0 (hunk 1 above)
Comment (danielrozenberg) [reply extensions/amp-ad-network-adzerk-impl/0.1/data/0:2]: Nope, but neither is this hosted on `cdn.ampproject.org` - it's a fake URL for a fake template

File: src/config.js (hunk 2 above)
Comment (danielrozenberg) [reply src/config.js:69]: I can do search & replace pretty easily but more than that requires more familiarity with the runtime than I have 😅

**Pull request closed without merging**